### PR TITLE
FC Diff view (FC section only) 

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

Screen Shot 2022-12-10 at 7 02 20 pm

Screen Shot 2022-12-10 at 7 02 16 pm

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails/FundingCycleListItem.tsx [hunk 1]
```diff
@@ -2,20 +2,46 @@ import { Tooltip } from 'antd'
...
+}) {
   const {
     theme: { colors },
   } = useContext(ThemeContext)
+  const highlight =
+    diffStatus === 'new'
+      ? `#AF${colors.background.failure.slice(1)}` // 69% opacity
+      : diffStatus === 'old'
+      ? `#AF${colors.background.success.slice(1)}`
+      : undefined
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails/FundingCycleListItem.tsx:22]: @johnnyd-eth now that we have tailwind, can you pls update these colors to use Tailwind colors?

**Comment by tomquirk** [comment]
@johnnyd-eth this is so huge! Sorry for the delay in review, looks all good i think, just one comment + conflicts to fix.

**Review by wraeth-eth — CHANGES_REQUESTED** [review]
LFG!!!! Love this feature. Just made some comments about tailwindcss stuff

File: src/components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails/FundingCycleListItem.tsx [hunk 2]
```diff
@@ -1,10 +1,46 @@
...
+      ? 'bg-error-100 dark:bg-error-900'
+      : diffStatus === 'new'
+      ? 'bg-success-100 dark:bg-success-900'
+      : undefined
+
   return (
-    <div className="ml-1 whitespace-nowrap text-grey-500 dark:text-grey-300">
-      {value}
+    <div
+      style={{
```
[the remark this answers is not in the document]
Comment (johnnyd-eth) [reply src/components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails/FundingCycleListItem.tsx:28]: Thank you ser, very helpful 💪

**Comment by johnnyd-eth** [comment]
Thanks for these tips @wraeth-eth, I've applied these changes! Good to go now?

**Review by wraeth-eth — APPROVED** [review]

**Pull request merged**